docs/ldml/tr35-keyboards.md

```diff
@@ -777,24 +783,40 @@ _Attribute:_ `width="1.2"` (optional, default "1.0")
 
 ##### Implied Keys
 
-Not all keys need to be listed explicitly.  The following keys can be assumed to already exist:
+Not all keys need to be listed explicitly.  The following two can be assumed to already exist:
 
 ```xml
-<!--  all 26 upper and lower case English letters -->
-<key id="a" to="a"/>
-<key id="b" to="b"/>
-<key id="c" to="c"/>
+<key id="gap" gap="true" width="1"/>
+<key id="space" stretch="true" width="1"/>
```

Comment:
```suggestion
<key id="space" to=" " stretch="true" width="1"/>
```

We should be a little careful with this one. Some keyboards override spacebar, e.g. Khmer keyboards currently emit ZWSP or COENG MARKER, not space, by default. We may need a paragraph:

> It is permissible to redefine the implied keys.